aws-datastore/src/test/resources/on-delete-request-for-blog-owner.txt

```diff
@@ -11,3 +12,6 @@ subscription OnDeleteBlogOwner {
     wea
   }
 }
+",
+  "variables": null
+}
```

Comment:
Does this need to be explicitly present and null? Or can we figure out how to omit this block from the output, if the value is null?

Reply:
I'm not sure - I was trying to maintain behavior of the existing code, in case AppSync does require it explicitly.  We can try to remove it later, after doing some more thorough end to end testing.